openfl-tutorials/interactive_api/Pytorch_Market_Re-ID/envoy_folder/two/market_shard_descriptor.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (C) 2020-2021 Intel Corporation
```

Comment:
There is no need to keep two identical Shard Descriptors. I suggest removing one/ and two/ folders under `envoy_folder` directory and keep just two different configs files

Reply:
ok, done